### ISSUE TITLE
enhancement: Mark functions that may panic automaticaly

### ISSUE THREAD

**Issue by Dushistov** [steveklabnik/rustdoc#78]
It would be nice if `rustdoc` can automaticaly calculate `is this function can cause panic` or not and append this info to generated documentation. Cause panic = there is function that call `panic!` in call tree with root in this function.

**Comment by steveklabnik** [comment]
I don't think this is the kind of thing we can do. Depends entirely on https://github.com/steveklabnik/rustdoc/issues/7

**Comment by steveklabnik** [comment]
Marking this as hard because it might even be impossible.
